src/components/Card.js

```diff
@@ -29,12 +71,19 @@ export default class Card {
     this._deleteButton = this._cardElement.querySelector(
       ".card__delete-button"
     );
+    this._deleteCardModal = document.querySelector(".delete-card-modal");
     this._likeButton = this._cardElement.querySelector(".card__like-button");
+    this._deleteCardModal
+      .querySelector(".modal__close-button")
+      .addEventListener("click", () => {
+        this._deleteCardModal.classList.remove("modal_opened");

```

Comment:
* Any modal should be created using a class, which extends `Popup` (it handles the overlay and `close icon`)